dts/bindings/pinctrl/atmel,sam-pinctrl.yaml

```diff
@@ -16,22 +16,6 @@ properties:
       const: 1
 
 child-binding:
-    description: atmel pins
```

Comment:
Are you able to leave `description:` here ?

Reply:
Fixed!